<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lbl_img_settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/menu_icons/Resources/settings_big.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_profil">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Profil 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Profil 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Profil 3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_profil">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil à modifier</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Modifier le profil</string>
    </property>
    <widget class="QLabel" name="lbl_code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Code télécommande</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_hd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Boîtier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_hd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>HD1 (Boîtier 1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD2 (Boîtier 2)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD3 (Boîtier 3)</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtfield_code">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
